File: core/src/main/java/vn/loitp/flickr/model/photosetgetlist/WrapperPhotosetGetlist.java

package vn.loitp.flickr.model.photosetgetlist;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class WrapperPhotosetGetlist {

    @SerializedName("photosets")
    @Expose
    private Photosets photosets;
    @SerializedName("stat")
    @Expose
    private String stat;

    public Photosets getPhotosets() {
        return photosets;
    }

    public void setPhotosets(Photosets photosets) {
        this.photosets = photosets;
    }

    public String getStat() {
        return stat;
    }

    public void setStat(String stat) {
        this.stat = stat;
    }

}

File: core/src/main/java/com/loitpcore/restApi/flickr/model/photoSetGetList/WrapperPhotosetGetlist.kt
package com.loitpcore.restApi.flickr.model.photoSetGetList

import androidx.annotation.Keep
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import com.loitp.core.base.BaseModel

/**
 * Created by Loitp on 04,August,2022
 * Galaxy One company,
 * Vietnam
 * +840766040293
 * freuss47@gmail.com
 */
@Keep
class WrapperPhotosetGetlist : BaseModel() {
    @SerializedName("photosets")
    @Expose
    var photosets: Photosets? = null

    @SerializedName("stat")
    @Expose
    var stat: String? = null
}
